esper/controllers/device/device.py

```diff
@@ -261,7 +297,10 @@ def set_active(self):
                     self.app.render(f'Device does not exist with name {device_name}\n')
                     return
                 response = search_response.results[0]
-                db.set_device({'id': response.id, 'name': response.device_name})
+                name = response.device_name
+                if response.alias_name and response.alias_name != '':
```

Comment:
You are using multiple times. Can you make it a function ?

Reply:
Done!